main.tf

```diff
@@ -1 +1,346 @@
+###
+### LOCALS
+###
 
+locals {
+  name_id         = "${var.name}-${random_string.this.result}"
+  bucket          = "vault-salt-formula"
+  ami_name_filter = "spel-minimal-centos-7-hvm-*.x86_64-gp2"
```

Comment:
these ami name locals ought to be variables. not everyone will want to use `spel`